source-academy/cadet-frontend
# Add Clarity for Assessment Source Version

### Description

This PR Fixes the issue of Source Chapter not being displayed for various Assessments in Source Academy. .

The main Implementation behind this is to add a simple `isClickable` flag in order to ensure that students **are not** allowed to change the Source Chapter of Assessments. 

Fixes #953

### Type of change


### How to Test

1. Login to Source Academy as any user. 
2. Go to an Assessment 
3. Ensure the Source Chapter matches with the `ChapterSelect` button 
4. `ChapterSelect` **should not** be clickable. 

### Checklist

Please delete options that are not relevant.
